test/ValidationFramework/Validation.hpp

```diff
@@ -0,0 +1,125 @@
+        std::vector<Comparison> comparisons_;
+    };
+
+    std::vector<TestCase> testCases_;
+
+   protected:
+    void SetUp() override
+    {
+        // List of validation test scenarios go here (replacement for a "super yaml file", althought this could be added
+        // in the future)
+
+        testCases_ = {
+            {
+                "001: Spherical gravity only",
+                {
+                    {
+                        Tool::GMAT,
+                        {
+                            {OutputQuantity::CARTESIAN_POSITION, 1.0e-3},
+                            {OutputQuantity::CARTESIAN_VELOCITY, 1.0e-6},
+                        },
+                    },
+                    {
+                        Tool::OREKIT,
+                        {
+                            {OutputQuantity::CARTESIAN_POSITION, 1.0e-3},
+                            {OutputQuantity::CARTESIAN_VELOCITY, 1.0e-6},
+                        },
+                    },
+                },
+            },
+            {
+                "002: EGM96 60x60 gravity",
+                {
+                    {
+                        Tool::GMAT,
+                        {
+                            {OutputQuantity::CARTESIAN_POSITION, 1.0e-3},
+                            {OutputQuantity::CARTESIAN_VELOCITY, 1.0e-6},
+                        },
+                    },
+                    {
+                        Tool::OREKIT,
+                        {
+                            {OutputQuantity::CARTESIAN_POSITION, 1.0e-3},
+                            {OutputQuantity::CARTESIAN_VELOCITY, 1.0e-6},
+                        },
+                    },
+                },
+            },
+            {
+                "003: Exponential Atmsohpere",
+                {
+                    {
+                        Tool::GMAT,
+                        {
+                            {OutputQuantity::CARTESIAN_POSITION, 1.0e-3},
+                            {OutputQuantity::CARTESIAN_VELOCITY, 1.0e-6},
+                        },
+                    },
+                    {
+                        Tool::OREKIT,
+                        {
+                            {OutputQuantity::CARTESIAN_POSITION, 1.0e-3},
+                            {OutputQuantity::CARTESIAN_VELOCITY, 1.0e-6},
+                        },
+                    },
+                },
+            },
+            {
+                "004: Constant thruster maneuver in In-Track direction",
+                {
+                    {
+                        Tool::GMAT,
+                        {
+                            {OutputQuantity::CARTESIAN_POSITION, 1.0e-3},
+                            {OutputQuantity::CARTESIAN_VELOCITY, 1.0e-6},
+                        },
+                    },
+                    {
+                        Tool::OREKIT,
+                        {
+                            {OutputQuantity::CARTESIAN_POSITION, 1.0e-3},
```

Comment:
Do we have standardized names for the output columns in csv in the different tools, yeah?